e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -652,3 +1087,11 @@ const clickHelper = buttonName => {
   cy.wait(100);
   cy.button(buttonName).click();
 };
+
+function getActionEditorModal() {
```

Comment:
nit: could the name be more consistent? e.g.
```suggestion
function getActionCreatorModal() {
```

so we would talk about only one thing e.g. `ActionCreator` not `ActionEditor`. FWIW, we seem to have `QueryActionEditor` which is just the text area part of the `ActionCreator`
![image](https://github.com/metabase/metabase/assets/1937582/25986e1b-5159-4c9d-a541-149de9cc44d1)